exercises.ipynb

```diff
@@ -172,22 +174,175 @@
+       "      <td>3</td>\n",
+       "      <td>14.126967</td>\n",
+       "      <td>SEK</td>\n",
+       "      <td>1.314161</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>4</td>\n",
+       "      <td>23.406904</td>\n",
+       "      <td>HKD</td>\n",
+       "      <td>2.678105</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>995</td>\n",
+       "      <td>13.593894</td>\n",
+       "      <td>HRK</td>\n",
+       "      <td>1.822751</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>996</td>\n",
+       "      <td>41.710860</td>\n",
+       "      <td>ZAR</td>\n",
+       "      <td>2.466347</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>997</td>\n",
+       "      <td>12.877760</td>\n",
+       "      <td>AUD</td>\n",
+       "      <td>7.980269</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>998</td>\n",
+       "      <td>29.561696</td>\n",
+       "      <td>KRW</td>\n",
+       "      <td>0.022690</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td>999</td>\n",
+       "      <td>42.034297</td>\n",
+       "      <td>RON</td>\n",
+       "      <td>8.838162</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>1000 rows × 3 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         value curr  in_euros_please\n",
+       "0    48.910052  THB         1.452716\n",
+       "1    16.505115  THB         0.490232\n",
+       "2    30.370579  INR         0.383891\n",
+       "3    14.126967  SEK         1.314161\n",
+       "4    23.406904  HKD         2.678105\n",
+       "..         ...  ...              ...\n",
+       "995  13.593894  HRK         1.822751\n",
+       "996  41.710860  ZAR         2.466347\n",
+       "997  12.877760  AUD         7.980269\n",
+       "998  29.561696  KRW         0.022690\n",
+       "999  42.034297  RON         8.838162\n",
+       "\n",
+       "[1000 rows x 3 columns]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import requests\n",
+    "import json\n",
+    "\n",
+    "# function one to get the latest rates out of the api\n",
+    "\n",
+    "def get_rates():\n",
+    "    latestrates = requests.get('https://api.exchangeratesapi.io/latest')\n",
+    "    return(json.loads(latestrates.content))\n",
+    "\n",
+    "# function two to get add a new column in euros through iteration\n",
+    "\n",
+    "def assets_df(csv):\n",
+    "    assetsdata = pd.read_csv(csv)\n",
+    "    rates = get_rates()['rates']\n",
+    "    new_column = []\n",
+    "   \n",
+    "    for x in assetsdata.iterrows():\n",
```

Comment:
looping through a list to create a new list of the same size... sounds like a map, no?